google-map-directions.html

```diff
@@ -83,6 +91,24 @@
       travelMode: 'DRIVING',
 
       /**
+       * Options for rendering.
+       *
+       * @attribute rendererOptions
+       * @type object
+       * @default {}
+       */
+      rendererOptions: {},
+
+      /**
+       * A panel for displaying directions
+       *
+       * @attribute panel
+       * @type HTMLElement
+       * @default null
+       */
+      panel: null,
```

Comment:
That's a fair point, and probably advantageous assuming only one `google-map-directions` element is present. I decided to add the attribute to avoid code duplication in [my own project](https://github.com/mikeraimondi/bike_share_directions/blob/frontendv1/frontend/elements/bsd-main/bsd-main.html#L98). Having the attribute available for binding allows me to more cleanly pass the same options to multiple `directions` elements.

NB: I'm somewhat new to Polymer. I may be missing something blindingly obvious.